github/issues.go

```diff
@@ -305,16 +310,29 @@ func (s *IssuesService) Edit(ctx context.Context, owner string, repo string, num
 	return i, resp, nil
 }
 
+// LockIssueOptions specifies the optional parameters to the
+// IssuesService.Lock method.
+type LockIssueOptions struct {
+	// LockReason specifies the reason to lock this issue.
+	// Providing a lock reason can help make it clearer to contributors why an issue
+	// was locked. Possible values are: off-topic, too heated, resolved, spam
```

Comment:
nit: add quotes for clarity and end in period:
`Possible values are: "off-topic", "too heated", "resolved", and "spam".`